src/modules/timestamp.ts

```diff
@@ -0,0 +1,3 @@
+import firebase from './firebase'
+
+export default firebase.firestore.Timestamp
```

Comment:
I think there is a yet better way to simplify that.
See, Timestamp is too general name.
And you locked it specifically for utils of Firebase.
And the Timestamp doesn't address it at all.
If you want to simplify imports, I would rather create a module for each function you want to export.
For example, for `Timestamp.now()` I would create the module `createFirebaseNowTimestamp`.
For `Timestamp.fromDate` - `createFirebaseTimestampFromDate`.